LEGO1/lego/legoomni/src/common/legogamestate.cpp

```diff
@@ -141,6 +141,18 @@ const char* g_strEnable = "enable";
 // STRING: LEGO1 0x100f3bf4
 const char* g_strDisable = "disable";
 
+inline LegoGameState::ScoreItem& LegoGameState::ScoreItem::operator=(const ScoreItem& p_other)
+{
+	// MSVC auto-generates an operator=, but LegoGameState::WriteScoreHistory() has a much better match
+	// with a manual implementation. Not sure if this function is supposed to go be in legogamestate.h
+	// instead of having an `inline` modifier.
+	m_totalScore = p_other.m_totalScore;
+	memcpy(m_scores, p_other.m_scores, sizeof(m_scores));
+	m_name = p_other.m_name;
+	m_playerId = p_other.m_playerId;
+	return *this;
+}
+
```

Comment:
Not sure if this should be `inline` or part of the header. Thoughts?

Reply:
I'd put it in the header